src/lib_json/json_value.cpp

```diff
@@ -1207,11 +1207,13 @@ bool Value::removeMember(const char* begin, const char* end, Value* removed) {
   if (it == value_.map_->end())
     return false;
   if (removed)
+  {
```

Comment:
Join with previous line to match surrounding brace style.

But other than adding helpful braces, this change doesn't appear to have changed the code's behavior.
The #if/#else code sections were single statements. In both the #if and the #else sections, adding braces around the section doesn't change its meaning.

Maybe I'm missing something?

Reply:
You right, I was at tag 1.8.3 and the fix you mention has been pushed later.
The PR is still valid for the test part that is as important as the check-for-null-pointer stuff.